src/cli/global/upgrade_all.rs

```diff
@@ -30,6 +30,10 @@ pub struct Args {
 
     #[clap(flatten)]
     config: ConfigCli,
+
+    /// The platform to install the package for.
+    #[clap(long, default_value_t = Platform::current())]
+    platform: Platform,
```

Comment:
On my machine this doesn't work if I use `emscripten-wasm32` 

Also wouldn't it make sense to by default get this from the environment itself? Just like the channels.

Reply:
@wolfv Could you create an wasm example to run some wasm stuff?

```toml
[project]
name = "wasm"
version = "0.1.0"
description = "Add a short description here"
authors = ["Wolf Vollprecht <w.vollprecht@gmail.com>"]
platforms = ["emscripten-wasm32", "osx-arm64"]
channels = ["conda-forge"]

[feature.host]
channels = ["conda-forge"]
platforms = ["osx-arm64"]
dependencies = { python = "*" }

[feature.wasm]
channels = ["https://repo.mamba.pm/emscripten-forge"]
dependencies = { python = "*", pillow = "*", numpy = "*" }
platforms = ["emscripten-wasm32"]

[environments]
wasm = ["wasm"]
host = ["host"]
```

Reply:
Let's do it after the merge so that Torsten can help :)